src/com/google/javascript/jscomp/AbstractCommandLineRunner.java

```diff
@@ -579,12 +604,47 @@ public FlagUsageException(String message) {
    *
    * @param files A list of filenames.
    * @param zips A list of zip filenames.
+   * @param jsonFiles A list of json encoded files.
+   * @return An array of inputs
+   */
+  protected List<SourceFile> createInputs(List<String> files,
+      List<String> zips, List<JsonFileSpec> jsonFiles) throws FlagUsageException, IOException {
```

Comment:
I definitely combine the json inputs with regular --js flag files. I just did the zip file because I wanted the option to still be there. I'm trying to keep as much parity with the command line runner as possible.

See https://github.com/ChadKillingsworth/closure-compiler-npm/blob/master/test/gulp.js#L206